### PR TITLE
Replace Button, Link, and Highlight with design system components.

### PR DESCRIPTION
**Pull request by leerob**
## Overview

This PR begins pulling out custom rolled components in favor of the Storybook Design System (SDS). 

### Buttons
**Prod**
image
**This PR**
image
Obviously not ideal. Added issue here -> https://github.com/storybookjs/design-system/issues/67

### Links
**Prod**

**This PR**

### Highlight
**Prod**
image

**This PR**
image

## Small Differences
**Prod**
image
**This PR**
image

**Prod**
image

**This PR**
image

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,211 +0,0 @@
-import React from 'react';
```
Comment (leerob) [inline src/components/Button.js]: 👋

File: src/components/Link.js [hunk 2]
```diff
@@ -1,105 +1,13 @@
...
-	}
-`;
-
-const LinkGatsby = LinkWrapper.withComponent(GatsbyLink);
+import { Link as DefaultLink } from '@storybook/design-system';
 
 function Link({ isGatsby, ...props }) {
   if (isGatsby) {
-    return <LinkGatsby {...props} />;
+    return <DefaultLink LinkWrapper={GatsbyLink} {...props} />;
```
Comment (leerob) [inline src/components/Link.js]: I opted to keep these file as a wrapper around `Link` to prevent having `LinkWrapper={GatsbyLink}` scattered everywhere.

**Comment by kylesuss** [comment]
Hi @leerob ! Thanks for the contribution here & over in the design system as well.

We have a PR in the works right now that totally replaces the duplicated components in this repo w/ the design system components as well as drastically alters the site hierarchy & design. You can see that here:

**Comment by kylesuss** [comment]
Hi @leerob ! Thanks for your interest in the project as well as the contributions here & over in the design system as well.

We have a similar PR in the works right now that totally replaces the duplicated components in this repo w/ the design system components as well as drastically alters the site hierarchy & design. You can see that here:

https://github.com/chromaui/learnstorybook.com/pull/106

We are just waiting to add some new content in that PR before we move forward with that merge. With that in mind, I think it makes sense to avoid duplicating this effort in smaller PRs as it will all be handled by #106 .

Sorry for the bad news! We have a design channel on the [Storybook discord](https://discord.gg/UUt2PJb) that you can hop on if you'd like to flesh out ideas as well. I certainly don't want you to have any wasted effort as we appreciate your help.

**Comment by leerob** [comment]
@kylesuss No worries - I definitely missed that PR! My bad. I'll give that PR a look over 😄 

I'll go ahead and close this 👍

**Comment by kylesuss** [comment]
Sounds good @leerob . Yeah, I think the PR could have used a more descriptive title so that it is obvious that the design system work is apart of it. I've gone ahead and updated it 👌

**Pull request closed without merging**